tests/FlatBuffers.Test/FlatBuffersExampleTests.cs

```diff
@@ -110,14 +110,20 @@ private void CanCreateNewFlatBufferFromScratch(bool sizePrefix)
                 Monster.FinishMonsterBuffer(fbb, mon);
             }
 
-
+#if ENABLE_SPAN_T
```

Comment:
If C# supports it, can we have these directives indented, here and elsewhere?

Reply:
I can indent them, but it seems like this would be inconsistent with the rest of the #ifdef's in the project.  Do you want me to indent every #ifdef?

Reply:
Ah I guess leave it for now then.